### PR TITLE
Fix Travis CI build following Python 3 switch

### PR DESCRIPTION
**Pull request by sbesson**
- Build Docker image with various OpenJDK base image
- Only build the components in the image to avoid exceeding build log limit

### CODE REVIEW

File: Dockerfile [hunk 1]
```diff
@@ -24,7 +24,7 @@ RUN pip install -r ome-model/requirements.txt
 RUN mvn clean install -DskipSphinxTests
 
 WORKDIR /bio-formats-build/bioformats
-RUN ant clean jars tools test
+RUN ant jars tools
```
Comment (joshmoore) [inline Dockerfile]: Will something else run tests in travis?

Comment (sbesson) [reply Dockerfile]: The `mvn` command above will run tests as part of the build lifecycle. Primary question is whether there is a 100% overlap with what `ant test` was executing and if not how we plan to solve this.

Comment (joshmoore) [reply Dockerfile]: Ah, ok. `skipSphinxTests` actually is `skipSphinx` not `skipSphinxAndTests`?

Comment (sbesson) [reply Dockerfile]: Yes - see https://github.com/ome/bio-formats-documentation/blob/master/pom.xml#L522, `skipSphinxTests` actually means skip the `linkcheck` step executed as part of the `test` phase of the Sphinx component.

Comment (jburel) [reply Dockerfile]: Renaming it to ``skipLinkcheck`` will make sense

File: .travis.yml [hunk 2]
```diff
@@ -1,22 +1,12 @@
...
-before_install:
-  - git submodule update --remote
-  - pip install --user -r ome-model/requirements.txt
+env:
+  - BUILD_IMAGE=openjdk:8
+  - BUILD_IMAGE=openjdk:11
 
 script:
-  - mvn install -DskipSphinxTests=true
+  - docker build -t bio-formats-build .
```
Comment (jburel) [inline .travis.yml]: add missing line

**Comment by sbesson** [comment]
Discussed earlier with the @ome/formats team. Proposing to get this change integrated and to start investigating the following todos

- uncap the Sphinx version across the documentation repos to fix the doc job
- review overlapping tests between Ant and Maven i.e. effectively start deprecating the `ant` build system at least up to the bundle creation
- review the Bio-Formats CI workflow to use this base Docker image more systematically

**Pull request merged**